scripts/plot.py

```diff
@@ -27,7 +27,15 @@
 import pandas as pd
 import plotly.plotly as py
 
-from plotly.graph_objs import *
+from plotly.graph_objs import (Bar,
```

Comment:
yikes, I'd break the first `Bar` onto a new line and then list all of the imported items on the same line, breaking only if you hit the end of the line.

alternatively, you could do `from plotly import graph_objs` and then later do `graph_objs.Bar`, etc.